project-base/src/Shopsys/ShopBundle/Resources/views/Front/Content/Product/filterFormMacro.html.twig

```diff
@@ -18,9 +18,9 @@
             <div
                 class="js-range-slider"
                 data-minimum-input-id="{{ filterForm.minimalPrice.vars.id }}"
-                data-minimal-value="{{ priceRange.minimalPrice }}"
+                data-minimal-value="{{ priceRange.minimalPrice.toValue }}"
```

Comment:
```suggestion
                data-minimal-value="{{ priceRange.minimalPrice.toString }}"
```

I think we should use `toString` right now, not later. The same is valid for maximalPrice

Reply:
We might want to use a Twig filter instead. Let's wait with this into the phase 4.